### PR TITLE
Add social actions

### LINKED ISSUE #9: Hook up social sharing actions

### LINKED ISSUE #10: Add social media links to header

### PR DESCRIPTION
**Pull request by nickcatal**
@jungshadow @jhjdesigns can you review this?

It borrows heavily from the previous social actions javascript code

It works, and I think it's done correctly, but considering I've never touched React before

noidea

Closes #9 
Closes #10

### CODE REVIEW

**Comment by jungshadow** [comment]
@nickcatal Don't worry…you're in good company ;)

I'll look at this when I get back to a computer. One quick note, I was thinking about moving `fucktify` to helpers.js, but it's fairly tied to addresses and needs to be refactored and made to be more flexible before I'd consider it a non-component function.

File: src/scripts/components/Actions.js [hunk 1]
```diff
@@ -13,35 +64,91 @@ import ReactDOM from 'react-dom';
...
+        const authorTwitter = helpers.shuffle([
+            'golovashkina',
+            'joshualturner',
+            'jungshadow',
+            'momaraqa',
+            'nickcatal',
+            'sixBcreative',
+        ]);
+
+        const text = encodeURI('I vote at ' + helpers.titlecase(helpers.fucktify(location.locationName)) + ' where the fuck do you vote? Find out at');
```
Comment (jungshadow) [inline src/scripts/components/Actions.js]: Ah...I get why you moved `fucktify`.

File: src/scripts/helpers.js [hunk 2]
```diff
@@ -7,7 +7,83 @@ let helpers = {
...
         }
+    },
+    /**
+     * Adds "fucking" in various places within a phrase, depending on the phrase
+     *
+     * @method fucktify
+     * @return {string} unsanitized phrase
+     */
+    fucktify: function(str) {
+    if (str === undefined) {
```
Comment (jungshadow) [inline src/scripts/helpers.js]: Is it me or is the indentation a bit off?

File: src/scripts/components/Actions.js [hunk 3]
```diff
@@ -2,8 +2,59 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import helpers from '../helpers';
 
 
+/**
```
Comment (jungshadow) [inline src/scripts/components/Actions.js]: This is largely a gut reaction (without thinking too much about it), but it seems weird for all of this stuff to be loaded just outside the component (lines 8-57) and not loaded elsewhere/independently.

**Comment by jungshadow** [comment]
LGTM. I don't have any showstopping concerns and, as long as it's performant, I'm ok with refining/refactoring as we go.

File: src/scripts/components/Actions.js (hunk 3 above)
Comment (nickcatal) [inline src/scripts/components/Actions.js]: I agree. I spent a bit of time trying to figure out a better way to go about it, played around with gulp a bit, etc. Couldn't figure anything out I felt comfortable with and didn't want that to be what holds up this.

If someone wants to refactor it I'd be quite happy :)

File: src/scripts/helpers.js (hunk 2 above)
Comment (nickcatal) [inline src/scripts/helpers.js]: Yep, fixed.

**Pull request merged**